src/bin/lockin.rs

```diff
@@ -566,9 +573,16 @@ mod app {
                     .inner_mut()]);
             });
 
-            c.local.usb_terminal.process().unwrap();
+            c.shared.settings.lock(|settings| {
+                c.local.usb_terminal.process(settings).unwrap();
```

Comment:
`process()` should return a bool whether the settings changed. Like miniconf's mqtt client does on `update()`.